sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -196,6 +210,29 @@ class RocksDB(
     }
   }
 
+  private def countKeys(): Long = {
```

Comment:
This doesn't leverage iterator() since here we don't need to deserialize the key-value from RocksDB to produce UnsafeRowPair.